src/MenuStyle.php

```diff
@@ -423,4 +472,156 @@ public function setTitleSeparator(string $actionSeparator) : self
 
         return $this;
     }
+
+    private function generateBorderRows() : void
+    {
+        $borderRow = sprintf(
+            "%s%s%s%s%s\n",
+            str_repeat(' ', $this->margin),
+            $this->getBorderColourCode(),
+            str_repeat(' ', $this->width),
+            $this->coloursResetCode,
+            str_repeat(' ', $this->margin)
+        );
+
+        $this->borderTopRows = [];
```

Comment:
You could do something like the following if you wanted: 

```php
$this->borderTopRows = array_fill(0, $this->borderTopWidth, $borderRow)
```
Your call, I don't mind :)

Reply:
Yep I'll do that good call